pkgs/top-level/python-packages.nix

```diff
@@ -28490,15 +28490,25 @@ in modules // {
 
   mps-youtube = buildPythonPackage rec {
     name = "mps-youtube-${version}";
-    version = "0.2.6";
+    version = "0.2.7.1";
 
     disabled = (!isPy3k);
 
+    # disabled due to error in loading unittest
+    # don't know how to make test from: <mps_youtube. ...>
+    doCheck = false;
+
+    # before check create a directory and redirect XDG_CONFIG_HOME to it
```

Comment:
The problem here is that there are multiple failures in the tests. First the permission problem which is discussed in #12591. This will be fixed by the lines below and second the error which is mentioned in the comment with doCheck. So this is the error which raises.

The preCheck is not necessary if doCheck = false but it is necessary if it is not.